jbpm-workitems/jbpm-workitems-webservice/src/main/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandler.java

```diff
@@ -397,6 +402,7 @@ public void executeWorkItem(WorkItem workItem,
                     }
                     new Thread(new Runnable() {
 
+                        @Override
```

Comment:
In this method, shouldn't these be also invoked in case of importLocation and importNamespace are not empty (in that if block that creates the client)?
```
  Collection<WorkItemHeaderInfo> headers = WorkItemHeaderUtils.getHeaderInfo(workItem);
  if (!headers.isEmpty()) {
         client.getRequestContext().put(Header.HEADER_LIST, headers.stream().map(
                                        this::buildHeader).collect(Collectors.toList()));
  }
```

Reply:
Refactoring done